File: app/src/main/java/com/przemyslawlusnia/vocabularycreator/funwithtests/ClockComponent.java
package com.przemyslawlusnia.vocabularycreator.funwithtests;

import com.przemyslawlusnia.vocabularycreator.core.di.AppScope;
import dagger.Component;

@AppScope
@Component(modules = {ClockModule.class})
public interface ClockComponent {

  void inject(TestActivity testActivity);

}

File: app/src/main/java/com/przemyslawlusnia/vocabularycreator/funwithtests/ClockComponent.kt
package com.przemyslawlusnia.vocabularycreator.funwithtests

import com.przemyslawlusnia.vocabularycreator.core.di.AppScope
import dagger.Component

@AppScope
@Component(modules = arrayOf(ClockModule::class))
interface ClockComponent {

    fun inject(testActivity: TestActivity)

}
